packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -433,7 +469,37 @@ class Shortcuts extends StatefulWidget {
       }
       return true;
     }());
-    return inherited?.notifier;
+    return inherited?.manager;
+  }
+
+  /// Returns the [Intent] matching the given key set.
+  ///
+  /// Returns an intent matching a set of pressed keys, as if a key had been
+  /// propagated to all of the [Shortcuts] widgets in the given context.  If
+  /// `keysPressed` is not supplied, uses the current set of pressed keys in
+  /// [RawKeyboard.keysPressed].
+  ///
+  /// This allows determining ahead of time which intent would be matched from a
+  /// set of pressed keys.
+  ///
+  /// Returns null if it reaches the root of the widget tree and no [Shortcuts]
+  /// widget's [ShortcutManager] has a matching [Intent] for the key set.
+  static Intent? matchingIntent(BuildContext context, {LogicalKeySet? keysPressed}) {
```

Comment:
Do we have a use case for this? As far as I can tell, it's not used in this PR? (which may also mean that it is missing a test 😝 ) 

.. or github's code review is just hiding where it is used from me..

Reply:
No, this is something I needed in my previous implementation, but it seemed useful. You're right though, it needs a test if we're going to keep it.

Reply:
I would probably just remove it until we have a use case for it to keep the API simple...

Reply:
I would probably just remove it until we have a use case for it to keep the API simple...